tests/test_integration.py

```diff
@@ -11,11 +11,11 @@
 
 import os
 
-import pandas as pd
+import helpers
 import networkx as nx
+import pandas as pd
 
 import dhnx
```

Comment:
This is a relative import of a module in /tests. Was not written very clean which leads to confusion. I made the relative import explicit in https://github.com/oemof/DHNx/pull/42/commits/6d898596eb44f0af16ca991dd67edf7bce15db8b.